src/mesh/mod.rs

```diff
@@ -213,35 +226,72 @@ impl ShapeMesh {
             }
         }
 
-
         Ok(())
     }
 
-    /// Returns the point on the shape's surface the camera is currently looking at
-    pub fn get_focus(&self, camera: &Camera) -> Option<Point3<f32>> {
+    /// Returns equally distributed points on the near_plane. The number of points
+    /// returned is focus_points².
+    pub fn get_focii(&self, camera: &Camera, focus_points: u8) -> Vec<Point3<f32>> {
+        // sets the amount of focus points that may cause a split event
```

Comment:
What does this comment refer to? Not the constants below, right? So it should probably be moved somewhere else...